config/app.config

```diff
@@ -9,17 +9,6 @@ monitored_directories:
 # also enables adding the runfolder-ready marker through the API.
 can_create_runfolder: False
 
-# An Illumina machine will write a RTAComplete.txt file when it has
-# finished the sequencing process. But when a user manually moves
-# the runfolder to the processing directory one can not guaranty
-# that all files have been moved when the RTAComplete.tx exist in
-# the new directory. The completed_marker_file can be changed to
-# another file, a file which for example will signal that the
-# transfer process has completed.
-completed_marker_file:
```

Comment:
One downside with removing this from the config is that we lose some customization. It's very convenient to be able to specify your own completed marker files, based on your setup. For example I know that MiSeqs come with some analysis software that can be run directly after sequencing https://www.illumina.com/systems/sequencing-platforms/miseq/products-services/miseq-reporter.html . In that case RTAComplete.txt might not be the last file to be written. However, this is not something we do at our site and I don't know if anyone else is using this service in this way. There might be no need for us to support this. What's your thoughts regarding this?

Reply:
This is a good point. The problem is, I didn't like the idea of having to manage six different variables in our config files when it boils down to RTAComplete.txt vs CopyComplete.txt, and for our purposes, this value will vary between instruments but never vary between environments. So I feel quite strongly that it belongs in code. Code is always easier to debug than config differences.

My take is that a) config files should only be used when there will be differences between servers/envs and b) we should build and maintain code according to our needs. That someone out there needs this particular MiSeq use case is purely theoretical (as far as I know?), and I assume if anyone else is using our code they are forking and heavily modifying it for their use anyhow.

Thoughts?

Reply:
Yeah, I think you make a valid point.